dk/dkcore320/initializer.bal

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Comment:
year should be 2025. Check other places as well for correct licence

Reply:
updated: [fd3af43](https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/pull/492/commits/fd3af439b71de60c909905eca4aa13b385a76b2f)